test/test_util/tpcc/workload_cached.cpp

```diff
@@ -0,0 +1,239 @@
+  auto curr = sqls_.emplace("delivery", std::vector<std::string>{});
+  std::vector<std::string> &vec = curr.first->second;
+  vec.emplace_back("SELECT NO_O_ID FROM \"NEW ORDER\" WHERE NO_D_ID = 1 AND NO_W_ID = 2 ORDER BY NO_O_ID LIMIT 1");
+  vec.emplace_back("DELETE FROM \"NEW ORDER\" WHERE NO_O_ID = 1 AND NO_D_ID = 2 AND NO_W_ID = 3");
+  vec.emplace_back("SELECT O_C_ID FROM \"ORDER\" WHERE O_ID = 1 AND (O_D_ID = 2 AND O_W_ID = 3)");
+  vec.emplace_back("UPDATE \"ORDER\" SET O_CARRIER_ID = 1 WHERE O_ID = 1 AND O_D_ID = 2 AND O_W_ID = 3");
+  vec.emplace_back("UPDATE \"ORDER LINE\" SET OL_DELIVERY_D = 1 WHERE OL_O_ID = 1 AND OL_D_ID = 2 AND OL_W_ID = 3");
+  vec.emplace_back("SELECT SUM(OL_AMOUNT) AS OL_TOTAL FROM \"ORDER LINE\" WHERE OL_O_ID=1 AND OL_D_ID=2 AND OL_W_ID=3");
+  vec.emplace_back(
+      "UPDATE CUSTOMER SET C_BALANCE = C_BALANCE+1, C_DELIVERY_CNT=C_DELIVERY_CNT+1 WHERE C_W_ID=1 AND C_D_ID=2 AND "
+      "C_ID=3");
+}
+
+void WorkloadCached::InitIndexScan() {
+  auto curr = sqls_.emplace("index_scan", std::vector<std::string>{});
+  std::vector<std::string> &vec = curr.first->second;
+  vec.emplace_back("SELECT NO_O_ID FROM \"NEW ORDER\" WHERE NO_D_ID = 1");
+  vec.emplace_back("SELECT NO_O_ID FROM \"NEW ORDER\" WHERE 1 < NO_D_ID");
+  vec.emplace_back("SELECT NO_O_ID FROM \"NEW ORDER\" ORDER BY NO_W_ID");
+  vec.emplace_back("SELECT NO_O_ID FROM \"NEW ORDER\" ORDER BY NO_W_ID DESC");
+  vec.emplace_back("SELECT NO_O_ID FROM \"NEW ORDER\" ORDER BY NO_D_ID");
+  vec.emplace_back("SELECT NO_O_ID FROM \"NEW ORDER\" ORDER BY NO_W_ID, NO_D_ID DESC");
+  vec.emplace_back("SELECT NO_O_ID FROM \"NEW ORDER\" WHERE NO_W_ID = 1 ORDER BY NO_W_ID");
+  vec.emplace_back("SELECT NO_O_ID FROM \"NEW ORDER\" WHERE NO_W_ID = 1 ORDER BY NO_W_ID LIMIT 15 OFFSET 445");
+}
+
+void WorkloadCached::InitNewOrder() {
+  auto curr = sqls_.emplace("neworder", std::vector<std::string>{});
+  std::vector<std::string> &vec = curr.first->second;
+  vec.emplace_back("SELECT C_DISCOUNT, C_LAST, C_CREDIT FROM CUSTOMER WHERE C_W_ID=1 AND C_D_ID=2 AND C_ID=3");
+  vec.emplace_back("SELECT W_TAX FROM WAREHOUSE WHERE W_ID=1");
+  vec.emplace_back("SELECT D_NEXT_O_ID, D_TAX FROM DISTRICT WHERE D_W_ID=1 AND D_ID=2");
+  vec.emplace_back("INSERT INTO \"NEW ORDER\" (NO_O_ID, NO_D_ID, NO_W_ID) VALUES (1,2,3)");
+  vec.emplace_back("UPDATE DISTRICT SET D_NEXT_O_ID = D_NEXT_O_ID + 1 WHERE D_W_ID = 1 AND D_ID = 2");
+  vec.emplace_back(
+      "INSERT INTO \"ORDER\" (O_ID, O_D_ID, O_W_ID, O_C_ID, O_ENTRY_D, O_OL_CNT, O_ALL_LOCAL) VALUES (1,2,3,4,0,6,7)");
+  vec.emplace_back("SELECT I_PRICE, I_NAME, I_DATA FROM ITEM WHERE I_ID=1");
+  vec.emplace_back(
+      "SELECT S_QUANTITY, S_DATA, S_DIST_01, S_DIST_02, S_DIST_03, S_DIST_04, S_DIST_05, S_DIST_06, S_DIST_07, "
+      "S_DIST_08, S_DIST_09, S_DIST_10 FROM STOCK WHERE S_I_ID=1 AND S_W_ID=2");
+  // vec.emplace_back("UPDATE STOCK SET S_QUANTITY = 1, S_YTD = S_YTD + 1, S_ORDER_CNT = S_ORDER_CNT + 1, S_REMOTE_CNT =
```

Comment:
Is there a particular reason why this query is commented out? Just curious.

Reply:
I actually don't know... I'll just revert this.